src/timecode_test_parse.rs

```diff
@@ -820,6 +820,23 @@ mod test {
             premiere_ticks: 15256200960000,
         }
     )]
+    // Less than 4 frames past a minute should parse OK
+    #[case::t00_01_00_04_f59_94_df(
+        ParseCase{
+            frames_sources: vec![
+                Box::new("00:01:01;03".to_string()),
+            ],
+            seconds_sources: vec![],
+            ticks_sources: vec![],
+            rate: rates::F59_94_DF,
+            seconds: Rational64::new(3662659, 60000),
+            frames: 3659,
+            timecode: "00:01:01;07".to_string(),
```

Comment:
I'm a little confused here, this timecode is different from the input value. Is this passing in your tests?